users/utils.py

```diff
@@ -0,0 +1,29 @@
+import Json, jwt
+
+from django.http            import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+
+from .models             import User
+from groundbnb.settings.local import LOCAL_SECRET_KEY, ALGORITHM
+
+def UserTokenDeco(func):
```

Comment:
`decorator`도 `nested function`이기 때문에 함수명을 아래와 같은 형태로 만들어주는게 좋습니다!
대문사는 class나 일반 상수에서만 사용합니다!

```suggestion
def login_required(func):
```

Reply:
넵 데코레이터 브랜치에서 수정하였습니다